torch/csrc/jit/codegen/cuda/scheduler.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <ATen/core/ivalue.h>
+#include <torch/csrc/jit/codegen/cuda/fusion.h>
+
+namespace torch {
+namespace jit {
+namespace fuser {
+
+struct TORCH_CUDA_API Scheduler {
+ public:
+  static c10::optional<std::tuple<int,int,int,int>>
```

Comment:
We are returning launch config here as plain integers, I understand this is mostly for testing purpose?

Reply:
Launch config should likely be put in the codegen IR (once we have the PR merged)
https://github.com/csarofeen/pytorch/pull/134/files#diff-a69a1e1a785f9973492d35462863a1e5R228-R235

Leaving it here just for reference.